lib/hanami/utils/string.rb

```diff
@@ -141,6 +147,7 @@ def capitalize
       #   string = Hanami::Utils::String.new 'hanami_utils'
       #   string.classify # => 'HanamiUtils'
       def classify
+        return self if classified?
```

Comment:
@choallin This looks like a workaround that doesn't fix the real problem in the following lines. Would you like to check why it doesn't work properly without adding this `#classfied?` guard?

---

More on the guard. Guards are useful if they are inexpensive. This is not the case: `#classified?` requires a lot of computations.